src/components/sbb-map-container/sbb-map-container.scss

```diff
@@ -0,0 +1,71 @@
+@use '../../global/styles' as sbb;
+
```

Comment:
There is a bug on mobile view that two scrollbars are appearing.

Reply:
@jeripeierSBB can you describe where you hit this issue?

Reply:
You need to test a device with pointer devices, or activate simulation of it on mac: 
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/29278232/229756529-2161dca4-1acb-4668-9a60-65700e94768d.png)